libmscore/accidental.h

```diff
@@ -118,7 +118,7 @@ class Accidental final : public Element {
       static const char* subtype2name(AccidentalType);
       static AccidentalType value2subtype(AccidentalVal);
       static AccidentalType name2subtype(const QString&);
-      static bool isMicrotonal(AccidentalType t)  { return t > AccidentalType::FLAT2; }
+      static bool isMicrotonal(AccidentalType t)  { return t > AccidentalType::NATURAL_SHARP; }
```

Comment:
Oh no... This is not the intention... The idea is to fix the name of isMicrotonal(). Its functionality should keep the same, otherwise natural-sharps/flats will be automatically converted to common sharps/flats.

Reply:
oops